addons/mail/static/tests/message/message.test.js

```diff
@@ -810,9 +809,9 @@ test("Reaction summary", async () => {
         const userId = pyEnv["res.users"].create({ partner_id });
         pyEnv["res.partner"].create({ name, user_ids: [Command.link(userId)] });
         await withUser(userId, async () => {
-            await click("[title='Add a Reaction']");
+            await click(".o-mail-Message-actions [title='Add a Reaction']");
             await click(".o-mail-QuickReactionMenu button", { text: "😅" });
-            await contains(".o-mail-MessageReaction", { text: `😅${idx + 1}` });
+            await waitFor(`.o-mail-MessageReaction:contains(😅\n${idx + 1})`, { timeout: 3000 });
```

Comment:
`waitFor` is used here because `contains()` had resolution with a 3 seconds timeout.

Reply:
Don't forget the exact match regex then.

Reply:
There is the `exact` parameter, which is used in `queryOne`. Too bad there is no `waitForOne`?